README.md

```diff
@@ -1,2 +1,98 @@
-# Hello-Microverse
-Hello Microverse Project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
```

Comment:
- [ ] You have done an excellent job of utilizing the [given template](https://github.com/microverseinc/readme-template) for the README file :+1:. However, it is mandatory to retain all sections of the README file and not remove any. Currently, the sections such as "Built With", "Live Demo" (which can be filled with "Coming soon"), "Getting started" and "Future features" are missing. Please add all these sections as required :+1:.
- [ ] Also you should fill every section with the required information :+1:

Reply:
I've implemented the modifications needed. Is there a possibility that you take a look first? or shall I request another code review directly?